basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/connections/EditConnectionTestCase.java

```diff
@@ -0,0 +1,79 @@
+package org.jboss.hal.testsuite.test.configuration.messaging.connections;
+
+
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.category.Standalone;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.page.config.MessagingPage;
+import org.jboss.hal.testsuite.test.configuration.messaging.AbstractMessagingTestCase;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.wildfly.extras.creaper.commands.foundation.online.SnapshotBackup;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * @author <a href="padamec@redhat.com">Petr Adamec</a>
+ */
+
+@RunWith(Arquillian.class)
+@Category(Standalone.class)
+public class EditConnectionTestCase extends AbstractMessagingTestCase {
+    private static final SnapshotBackup snapshotBackup = new SnapshotBackup();
+
+
+    @Page
+    private MessagingPage page;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        client = ManagementClientProvider.createOnlineManagementClient();
+        client.apply(snapshotBackup.backup());
+        administration.reloadIfRequired();
+    }
+
+    @Before
+    public void before() {
+        page.viewConnectionSettings("default");
+        page.switchToConnectionFactories();
+    }
+
+    @AfterClass
+    public static void tearDown() throws IOException, OperationException, CommandFailedException, TimeoutException, InterruptedException {
+        client.apply(snapshotBackup.restore());
+        administration.reloadIfRequired();
+        client.close();
```

Comment:
I suggest to clean up the `client` in the same class where it is created - in the parent `AbstractMessagingTestCase`.